assets/src/blocks/CarouselHeader/EditableBackground.js

```diff
@@ -4,6 +4,31 @@ import { Button, Dropdown } from '@wordpress/components';
 import { toSrcSet } from './CarouselHeaderEditor';
 const { __ } = wp.i18n;
 
+const customCdnUrl = !window.p4ge_vars.stateless_url;
```

Comment:
This probably needs further checks to see if the custom URL is actually enabled.